experimental/auth-logic/wrappers/rekor_wrapper.go

```diff
@@ -0,0 +1,131 @@
+// describing that verifying the log entry entails:
+//  -- verifying the signature in `signedEntryTimestamp`, using Rekor's public key,
+// -- verifying the signature in `body.RekordObj.signature`, using Oak's public key,
+// -- verifying that the content of the body matches the input `endorsement_bytes`.
+// And also possibly validating the inclusion proof
+//
+// TODO look into validating the inclusion proof with
+// logEntryAnon.Verification.ValidateInclusionProof.
+// I'm not sure what to pass for "formats" yet
+// https://github.com/sigstore/rekor/blob/e0b79164f2279ad7c3e723a3ee16afbfcf271188/pkg/generated/models/log_entry.go#L365
+
+func getLogEntryAnonFromFile(rekorLogFilePath string) (*models.LogEntryAnon, error) {
+	// get LogEntry, which is a map from strings to LogEntryAnons
+	logEntryBytes, err := ioutil.ReadFile(rekorLogFilePath)
+	if err != nil {
+		return nil, fmt.Errorf("could not read the rekor log file: %v", err)
+	}
+
+	var logEntry models.LogEntry
+
+	err = json.Unmarshal(logEntryBytes, &logEntry)
+	if err != nil {
+		return nil, fmt.Errorf("could not unmarshal the logEntry from file: %s, err: %v", rekorLogFilePath, err)
+	}
+
+	if len(logEntry) != 1 {
+		return nil, fmt.Errorf("For transparent release, Rekor log entries should contain exactly one UUID: %v", logEntry)
+	}
+
+	var logEntryAnon models.LogEntryAnon
+	// set logEntryAnon to the only value in LogEntry (which is a map)
+	for _, anon := range logEntry {
+		logEntryAnon = anon
+	}
+	return &logEntryAnon, nil
+}
+
+func getRekordEntryFromAnon(logEntryAnon models.LogEntryAnon) (*rekord.V001Entry, error) {
+	bodyString, ok := logEntryAnon.Body.(string)
+	if !ok {
+		return nil, fmt.Errorf("could not coerce LogEntryAnon into string. LogEntryAnon: %v", logEntryAnon)
+	}
+
+	bodyDecoded, err := base64.StdEncoding.DecodeString(bodyString)
+	if err != nil {
+		return nil, fmt.Errorf("could not decode body from base64 %v: %v", logEntryAnon, err)
+	}
+
+	proposedEntry, err := models.UnmarshalProposedEntry(bytes.NewReader(bodyDecoded), runtime.JSONConsumer())
+	if err != nil {
+		return nil, fmt.Errorf("could not unmarshal proposed entry from body: %v, %s", bodyDecoded, err)
+	}
+
+	entryImpl, err := types.NewEntry(proposedEntry)
+	if err != nil {
+		return nil, fmt.Errorf("could not convert ProposedEntry into NewEntry: %v, %s", proposedEntry, err)
+	}
+	// rekord
+	rekordEntry, ok := entryImpl.(*rekord.V001Entry)
+	if !ok {
+		return nil, fmt.Errorf("could not convert NewEntry into rekord. NewEntry: %v,", entryImpl)
+	}
+	return rekordEntry, nil
+}
+
+// Verify signature in a rekord entry. In the context where this is used,
+// this will verify the contents of a rekord entry (an endorsement file)
+// against the product team's public key.
+func verifyRekordLogSignature(rekordEntry *rekord.V001Entry) error {
+	publicKey := rekordEntry.RekordObj.Signature.PublicKey.Content
+	// The unused argument is for extra bytes, not an error
+	block, _ := pem.Decode(publicKey)
```

Comment:
Does this work (so it's self documenting instead of needing a comment), or does Go complain about unused var?

```suggestion
	block, _bytes := pem.Decode(publicKey)
```